### PR TITLE
v8.1.2: dynamic config add `import.num-threads`

### PR DESCRIPTION
**Pull request by Oreoxmt**
### What is changed, added or deleted? (Required)

v8.1.2: dynamic config add `import.num-threads`

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v8.5 (TiDB 8.5 versions)
- [ ] v8.4 (TiDB 8.4 versions)
- [ ] v8.3 (TiDB 8.3 versions)
- [ ] v8.2 (TiDB 8.2 versions)
- [x] v8.1 (TiDB 8.1 versions)
- [ ] v7.5 (TiDB 7.5 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s): https://github.com/tikv/tikv/issues/17807

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: dynamic-config.md [hunk 1]
```diff
@@ -227,6 +227,7 @@ show warnings;
 | storage.flow-control.soft-pending-compaction-bytes-limit | 触发流控机制开始拒绝部分写入请求的 KvDB pending compaction bytes 阈值 |
 | storage.flow-control.hard-pending-compaction-bytes-limit | 触发流控机制拒绝所有新写入请求的 KvDB pending compaction bytes 阈值 |
 | storage.scheduler-worker-pool-size | Scheduler 线程池中线程的数量 |
+| import.num-threads | 处理 RPC 请求的线程数量（自 v8.1.2 起支持） |
```
Comment (3pointer) [inline dynamic-config.md:230]:
```suggestion
| import.num-threads | 处理恢复或导入 RPC 请求的线程数量（自 v8.1.2 起支持） |
```

**Comment by Oreoxmt** [comment]
@3pointer PTAL again

File: dynamic-config.md [hunk 2]
```diff
@@ -227,6 +227,7 @@ show warnings;
 | storage.flow-control.soft-pending-compaction-bytes-limit | 触发流控机制开始拒绝部分写入请求的 KvDB pending compaction bytes 阈值 |
 | storage.flow-control.hard-pending-compaction-bytes-limit | 触发流控机制拒绝所有新写入请求的 KvDB pending compaction bytes 阈值 |
 | storage.scheduler-worker-pool-size | Scheduler 线程池中线程的数量 |
+| import.num-threads | 处理恢复或导入 RPC 请求的线程数量（自 v8.1.2 起支持） |
 | backup.num-threads | backup 线程的数量（自 v4.0.3 起支持） |
```
Comment (Oreoxmt) [inline dynamic-config.md:231]:
```suggestion
| import.num-threads | 处理恢复或导入 RPC 请求的线程数量（自 v8.1.2 起支持在线修改） |
| backup.num-threads | backup 线程的数量（自 v4.0.3 起支持在线修改） |
```

File: dynamic-config.md [hunk 3]
```diff
@@ -227,6 +227,7 @@ show warnings;
 | storage.flow-control.soft-pending-compaction-bytes-limit | 触发流控机制开始拒绝部分写入请求的 KvDB pending compaction bytes 阈值 |
 | storage.flow-control.hard-pending-compaction-bytes-limit | 触发流控机制拒绝所有新写入请求的 KvDB pending compaction bytes 阈值 |
 | storage.scheduler-worker-pool-size | Scheduler 线程池中线程的数量 |
+| import.num-threads | 处理恢复或导入 RPC 请求的线程数量（自 v8.1.2 起支持） |
```
Comment (Oreoxmt) [inline dynamic-config.md:230]:
```suggestion
| import.num-threads | 处理恢复或导入 RPC 请求的线程数量（自 v8.1.2 起支持在线修改） |
```

**Review by 3pointer — APPROVED** [review]

**Pull request merged**